pkg/services/rpcsrv/server.go

```diff
@@ -2348,7 +2348,7 @@ func (s *Server) traverseIterator(reqParams params.Params) (any, *neorpc.Error)
 	session, ok := s.sessions[sID.String()]
 	if !ok {
 		s.sessionsLock.Unlock()
-		return []json.RawMessage{}, nil
+		return nil, neorpc.ErrUnknownSession
```

Comment:
`docs/rpc.md` as well.

Reply:
done. Let me know please if you want to combine this commit with doc edition